salt/modules/state.py

```diff
@@ -1377,6 +1377,19 @@ def sls_id(id_, mods, test=None, queue=False, **kwargs):
         :conf_minion:`pillarenv` minion config option nor this CLI argument is
         used, all Pillar environments will be merged together.
 
+    pillar
+        Custom Pillar values, passed as a dictionary of key-value pairs
+
+        .. code-block:: bash
+
+            salt '*' state.sls_id my_state my_module pillar='{"foo": "bar"}'
+
+        .. note::
+            Values passed this way will override Pillar values set via
```

Comment:
This doesn't seem clear enough. Will it only override a particular key if it is present or will it completely replace the previous pillar and remove even unrelated keys?